lib/hanyutils.ex

```diff
@@ -59,6 +77,35 @@ defmodule Hanyutils do
     |> Pinyin.marked()
   end
 
+  @doc """
+  Convert a string containing Han characters to Zhuyin.
+
+  For more information about `converter`, please refer to `Hanzi.to_pinyin/2`.
+  Because the Unihan database provides only definitions of pinyin
```

Comment:
Similarly as `to_marked_pinyin` and `to_numbered_pinyin` there's a `converter` parameter that allows choosing the pronounciations and defaults to `Hanzi.common_pronunciation`.
So yes, because we get the pronounciations from the Hanzi via Pinyin and it's configurable, the consumer should know.